ansible-ipi-install/roles/installer/tasks/57_check_vip_status.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Test the connectivity for {{ apivip }} 
```

Comment:
Can we change this to "Test for existing API VIP connectivity" -- Lets not use the variable {{ apivip }} because that will show an IP and the user may not be aware directly this is what we are testing.